tests/actions.go

```diff
@@ -3003,8 +3003,9 @@ func (oa *operatorActions) DeployIncrementalBackup(from *TidbClusterConfig, to *
 	isv1 := from.OperatorTag == "v1.0.0"
 
 	setString := map[string]string{
-		"binlog.pump.storage": "1Gi",
-		"binlog.pump.image":   fmt.Sprintf("pingcap/tidb-binlog:%v", from.ClusterVersion),
+		"binlog.pump.storageClassName": "standard",
```

Comment:
Is it OK that we do not set the sc explicitly so that the cases will use the default sc in different providers?